app/components/trainees/record_header/view.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Trainees
+  module RecordHeader
+    class View < GovukComponent::Base
+      attr_reader :trainee
+
+      def initialize(trainee:)
+        @trainee = trainee
+      end
+
+      delegate :trn, to: :trainee
+
+      def show_trn?
+        trn.present?
+      end
+
+      def name
```

Comment:
If you `include TraineeHelper` at the top of this class, you can remove this `name` method and use the traineer helper method `trainee_name(trainee)` directly in the component's html file.

Reply:
Thank you, done